src/batch-client/index.ts

```diff
@@ -193,6 +195,12 @@ export class ZimbraBatchClient {
 		});
 	};
 
+	public createMountpoint = (_options: CreateMountpointInput) =>
```

Comment:
Don't need to change it unless you are there for something else, but not sure why we are prefixing `_` for `_options` here...